testfixtures.go

```diff
@@ -89,21 +85,33 @@ func newContext(db *sql.DB, helper Helper, fixtures []*fixtureFile) (*Context, e
 	return c, nil
 }
 
+// CheckDatabaseName returns nil if databaseName matches regexp
+//     if err := fixtures.CheckDatabaseName(); err != nil {
+//         log.Fatal(err)
+//     }
+func (c *Context) CheckDatabaseName() error {
+	if skipDatabaseNameCheck {
+		return nil
+	}
+	dbName, err := c.helper.databaseName(c.db)
+	if err != nil {
+		return err
+	}
+	dbnameRegexp := regexp.MustCompile("(?i)test")
```

Comment:
This regexp should be a global variable right above the function definition (to avoid recompiling the regexp each time). Keep it unexported